examples/error_handling.rs

```diff
@@ -89,11 +89,10 @@ fn main() {
                 })
         })
         .and_then(|(bot, msg, file_id)| {
-            bot.photo(msg.chat.id).file_id(file_id).send().map_err(
-                |err| {
-                    (bot, msg, PhotoErr::Telegram(err))
-                },
-            )
+            bot.photo(msg.chat.id)
+                .file_id(file_id)
+                .send()
+                .map_err(|err| (bot, msg, PhotoErr::Telegram(err)))
```

Comment:
Nice! I don't know anymore why I have structured this function in such an ugly code style.